src/style.css

```diff
@@ -0,0 +1,101 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 14px;
+  margin: 0;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  padding-top: 16px;
+  padding-left: 5px;
+  padding-bottom: 35px;
+
+}
+
+.card__description {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-bottom: 5px;
+}
+
+.card__item__number,
+.card__rating__text,
+.card__price__box__text,
+.card__rating__review {
+  font-style: normal;
+  font-size: 11px;
+  line-height: 13px;
+  margin: 0 0 16px;
+}
+
+.card__price__box {
+  display: flex;
+  justify-content: space-between;
+  margin: 10px 0 5px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__rating_star {
+  display: flex;
+  margin-top: 1px;
+  margin-left: 1px;
+}
+
+.card__rating__item {
+  background-image: url(/images/star.svg);
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+}
+
+.card__rating .card__rating__item:nth-child(-n+4) {
```

Comment:
use styles from previous task, so when I change modifier - count of active stars should change accordingly

Reply:
fix this, use your code from previous task, add styles for all cases